cmd/manager/main.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/kudobuilder/kudo/pkg/controller/instance"
 	"github.com/kudobuilder/kudo/pkg/controller/operator"
 	"github.com/kudobuilder/kudo/pkg/controller/operatorversion"
+	util "github.com/kudobuilder/kudo/pkg/test/utils"
```

Comment:
1. IMO we should not import from the `pkg/test` packages.  This could should move out to a common lib location used by kudo and test
2. it should not be in a `utils` package.  IMO it looks like we have a code in `test/utils/` which should be in our project `kube` or `kubernetes` package.

Reply:
we have a number of kube Clients like this in `pkg/kubdoctl/kube/config.go`